docker/Dockerfile

```diff
@@ -73,10 +71,6 @@ RUN apt-get update && apt-get install -y \
     python3-nose \
     python3-sphinx
 
-# RUN wget -q https://github.com/Kitware/CMake/releases/download/v3.18.3/cmake-3.18.3-Linux-x86_64.sh && \
-#     chmod +x ./cmake-3.18.3-Linux-x86_64.sh && \
-#     cmake-3.18.3-Linux-x86_64.sh
```

Comment:
We might as well remove these lines from the docker file, since they are commented out.